src/packages/client/src/__tests__/benchmarks/huge-schema/builder.ts

```diff
@@ -0,0 +1,101 @@
+// This file is used to build large schemas for benchmarking purposes
+
+import fs from 'fs'
+import path from 'path'
+
+const numberToBase26 = (val: number, tail = '') => {
+  if (val <= 26) {
+    return `${String.fromCharCode(val + 64)}${tail}`
+  }
+
+  const remainder = val % 26 || 26
+  const division = Math.trunc(val / 26) - (remainder === 26 ? 1 : 0)
+
+  return numberToBase26(
+    division,
+    `${String.fromCharCode(remainder + 64)}${tail}`,
+  )
+}
+
+function write(location: string, data: string) {
+  if (fs.existsSync(location)) {
+    fs.unlinkSync(location)
+  }
+  fs.writeFileSync(location, data, {})
+}
+
+class Model {
+  name: string
+  body: string
+  constructor({ name, body }: { name: string; body?: string }) {
+    this.name = name
+    this.body = body ?? ``
+  }
+  public build() {
+    return `
+model ${this.name} {
+  ${this.body}
+}
+`
+  }
+}
+
+function main() {
+  let schema = `
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["nativeTypes", "groupBy"]
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+  }
+  
+  `
+  const ts = `
+import { PrismaClient } from  '@prisma/client'
+const client = new PrismaClient();
+
+async function main(){
+  const a = await client.a.findMany()
+}
+main().catch(err => console.log(err))
+`
+  const modelMap = new Map<string, Model>()
+  for (let i = 1; i < 50; i++) {
+    const modelName = numberToBase26(i).toLowerCase()
```

Comment:
What don't we just have a loop that generates `Model1` `Model2` etc? This looks a bit complicated 😄

Reply:
Because I wasn't thinking I guess :face_with_head_bandage: